docs/user/upgrade.rst

```diff
@@ -126,8 +126,14 @@ has been replaced with standard ingest type inference, which works similarly but
 Generally, the previous behavior can be replicated by using type inference to create a converter definition,
 then modifying the converter to set the feature ID to the first column (``$1``).
 
-Scalatra Version Updates
-------------------------
+Arrow Version Update
+--------------------
+
+The version of Apache Arrow used for Arrow-encoded results has been updated from 0.6.0 to 0.10.0. Due to changes
+in the Arrow inter-process communication (IPC) format, clients may need to update to the same Arrow version.
```

Comment:
May?;)

Reply:
I didn't have any luck with using the java 0.6 client, but @theneuralbit said the javascript client worked fine with both...

Reply:
Yeah there were some breaking changes to the format between 0.6 and 0.9 (not sure about 0.10), but the JS-0.3 release _should_ be able to handle it.